app/dashboard/templates/shared/sidebar_search.html

```diff
@@ -82,6 +83,7 @@
         </div>
       </div>
     </div>
```

Comment:
Could we we indent this component by 1 ?
( Cause it's within an `if` condition )

Reply:
yes, my formatting tool doesnt recognize this syntax.